src/inmanta/module.py

```diff
@@ -134,7 +136,7 @@ def __hash__(self) -> int:
 
     @property
     def specs(self) -> Sequence[tuple[str, str]]:
```

Comment:
We could probably phase this out if we wanted. But I'll leave the decision up to you.